pkg/scheduler/scheduler.go

```diff
@@ -420,6 +473,12 @@ func (s *Scheduler) scheduleNext() bool {
 	case ReconcileSchedule: // share same logic with first schedule
 		err = s.scheduleOne(key.(string))
 		klog.Infof("Reschedule binding(%s) as placement changed", key.(string))
+	case ResourceSchedule: // share same logic with first schedule
+		err = s.scheduleOne(key.(string))
+		klog.Infof("Reschedule binding(%s) as resource of workload changed", key.(string))
+	case ScaleSchedule: // share same logic with first schedule
+		err = s.scaleScheduleOne(key.(string))
```

Comment:
It means when replicas or request CPU/Memory changed in deployment template, the reference binding need to be rescheduled, right?

Reply:
ScaleSchedule only represent the condition that replicas changed. the change of request CPU/Memory is another type needs to be considered when scheduling with the ResourcesPreferences.